mods/ra/weapons/missiles.yaml

```diff
@@ -75,93 +36,87 @@ Dragon:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+^AntiGroundMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Ground, Water
+
+^AntiAirMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Air
+
+Maverick:
+	Inherits: ^AntiGroundMissile
+	ReloadDelay: 30
+	Range: 9c0
+	MinRange: 3c0
+	Report: missile7.aud
+	Burst: 2
+	BurstDelay: 7
+	Projectile: Missile
+		Speed: 256
+		Inaccuracy: 512
+		CruiseAltitude: 2c0
+		RangeLimit: 14c410
+	Warhead@1Dam: SpreadDamage
```

Comment:
Reading https://github.com/OpenRA/OpenRA/pull/12633/commits/2bb7b188d1043de0993cb34d115a8e41b9881b40#r98301340 I guess this is intentional.